control-plane/rest/openapi-specs/v0_api_spec.yaml

```
@@ -319,6 +319,79 @@ paths:
           $ref: '#/components/responses/ServerError'
       security:
         - JWT: []
+  '/nodes/{id}/label/{label}':
```

As discussed let's split this into key/value pair, so it matches up with create volume. (can be new pr)
Then if we add later labels(plural), then we just add a hashmap via body

```suggestion
  '/nodes/{id}/label/{key}={value}':
```

---

The openapi errors out stating `Declared path parameter key}={value needs to be defined as a path parameter in path or operation level` 
The reason being `=` is considered a query param, 

I tried this as well ` '/nodes/{id}/label/{key}/{value}':` , but IMHO it didn't looked good, 
1.  there is a single api for put and delete. And with this approach i need to pass `""` for delete as only key is required for deletion .

---

Ah seems our openapi generator has a bug leave this with me I'll fix it and update openapi